web/src/pages/HomeInfo/index.tsx

```diff
@@ -503,12 +506,25 @@ function HomeSloganCard({ selectedNetwork, config }) {
       </div><hr></hr>
       <HomeInfoActionButton>
         <HomeInfoExploreButton />
+        <HomeInfoCreateButton />
       </HomeInfoActionButton>
     </div></>)
 }
 
 
 function HomeInfoInstallCard({ selectedNetwork }) {
+  
+  const hasNotificationPermissions = useGlobalStore(
+    (state: GlobalState) =>
+      state.activities.notificationsPermissionGranted,
+  );
+
+  const [prompt, promptToInstall] = useAddToHomescreenPrompt();
```

Comment:
why are you copying code from other file to here? in a file which already has more then 500 lines?

Reply:
tried to check for a property from the other component to display the the InstallCard in homeinfo

Reply:
what? which property? from which component? to which component?

Reply:
I want to check that either Allow Notification button or Install App buttons are shown. So I need to check if the state of their components